### DISCUSSION TITLE
工单有不符合规则的报错依然可以强制提交

### DISCUSSION THREAD

**Discussion by planet-zjx** [hhyo/Archery#2064] [General]
如果工单有不符合规则的报错，怎样设置不能强制提交。

**Comment by nick2wang** [comment]
设置一下参数AUTO_REVIEW_WRONG

**Reply by planet-zjx** [reply]
你好，这个参数在哪个标签下，值是0和1吗？还是T和F

**Reply by nick2wang** [reply]
> 你好，这个参数在哪个标签下，值是0和1吗？还是T和F

设置成1，有警告就会驳回
![image](https://user-images.githubusercontent.com/33473924/221803785-4e29c537-30b6-4438-a3b4-12b2f6b99d65.png)

**Comment by planet-zjx** [comment]
多谢大佬
